src/components/Content/Content.js

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+
+const Content = (props) => {
+  return (
+    <>
+      <div className={`section ${props.className}`}>
+        <div className="component-section-header">
+          <div className="title-badge">
+            <div className="title-badge-title">{props.sectionTitle}</div>
+          </div>
+
+          <h2>
+            <div className="component-title ">{props.sectionSubtitle}</div>
+          </h2>
+
+          <div className="description">{props.description}</div>
+        </div>
+
+        <div className="component-usp-row">
+          {props.images.map((image, index) => (
+            <div className="item team" key={index}>
+              <img className="image" src={image.imageSrc} alt="images" />
+              <div className="component-title title">{image.imageTitle}</div>
+              {image.imageDescription ? (
+                <div className="description">{image.imageDescription}</div>
+              ) : null}
```

Comment:
When you write a ternary and one of the returns is `null` you can always rewrite it with `&&`s and make it shorter. In this case I would go for:
```js
{image.imageDescription && <div className="description">{image.imageDescription}</div>}
```

this is because **and** (&&) evaluates every expression and returns the last one if all the previous ones are `true`